src/app/utils/firebase.js

```diff
@@ -577,18 +580,22 @@ const FireBaseTools = {
       }
     );
 
-  },
+    },
 
 /**
   *  Populate the date event object with data from the Firebase Database
   * @param course snapshot from Firebase of the retrieved course section
   * @param date the date event array that will be pushed as a class event
   */
 setDateEvents : (course, date) => {
-
              let teacher = (course['First Name']+" "+course.Last);
              let room=(course['Room Nbr']);
-             let courseTime=(course['Mtg Start']+" - "+course['Mtg End']);
+             let defaultStartString = '12:00:0 AM';
+             let defaultEndString = '12:01:0 AM';
+             let startTime = course['Mtg Start']? course['Mtg Start'].split(":") : defaultStartString.split(":");
```

Comment:
Just to clarify, the defaultStartString and defaultEndString are only used if there is no start and end time. We needed to put a value rather than null so that we cant split and splice it.